code/game/objects/items/weapons/power_cells.dm

```diff
@@ -95,7 +95,7 @@
 	name = "infinite-capacity power cell!"
 	icon_state = "icell"
 	origin_tech =  null
-	maxcharge = 30000
+	maxcharge = 1000000000 //Good luck charging this, buff it if it gets fully charged
```

Comment:
Yeah, it was a joke about how arbitrarily large number is much smaller than the number you want

I am amused about the defines discussion though